providers/implementations/storemgmt/winstore_store.c

```diff
@@ -23,7 +23,9 @@
 #include "prov/implementations.h"
 #include "prov/bio.h"
 #include "file_store_local.h"
-
+#ifdef __CYGWIN__
+#include <Windows.h>
```

Comment:
Nit: should this be capitalized?

Reply:
Just to check capitalization doesn't matter on Windows :-) Fund aside: Changed in https://github.com/openssl/openssl/commit/f09f774fbb35f03866f0e463a815032c1b7d7375. Will be disappointed if it fails...